test/EVMHost.cpp

```diff
@@ -266,7 +266,7 @@ evmc::result EVMHost::call(evmc_message const& _message) noexcept
 			bytes(begin(message.sender.bytes), end(message.sender.bytes)) +
 			bytes(begin(message.create2_salt.bytes), end(message.create2_salt.bytes)) +
 			keccak256(bytes(message.input_data, message.input_data + message.input_size)).asBytes()
-		));
+		), h160::AlignLeft);
```

Comment:
These in #13533 triggered the change. It is error prone having this implicitly, and the refactor revealed a lot of code duplication around `FunctionSelector`.